docs/sources/explore/_index.md

```diff
@@ -63,4 +63,4 @@ After you've navigated to Explore, you should notice a "Back" button in the Expl
 
 > **Note:** Available in Grafana 7.3 and later versions.
 
-The Share shortened link capability allows you to create smaller and simpler URLs of the format /goto/:uid instead of using longer URLs with query parameters. To create a shortened link, click the **Share** option in Explore toolbar. Any shortened links that are never used will be automatically deleted after 7 days.
+The Share shortened link capability allows you to create smaller and simpler URLs of the format /goto/:uid instead of using longer URLs with query parameters. To create a shortened link to the executed query, click the **Share** option in Explore toolbar. Any shortened links that are never used will be automatically deleted after 7 days.
```

Comment:
```suggestion
The Share shortened link capability allows you to create smaller and simpler URLs of the format /goto/:uid instead of using longer URLs with query parameters. To create a shortened link to the executed query, click the **Share** option in the Explore toolbar. A shortened link that is never used will automatically get deleted after seven (7) days.
```